domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/registerdata/infotrygd/InfotrygdgrunnlagYtelseMapper.java

```diff
@@ -81,7 +91,7 @@ private static boolean skalMappeInfotrygdandeler() {
             .medUtbetalingsgrad(utbetalingsgrad)
             .medInntektskategori(i)
             .medRefusjonsgrad(BigDecimal.ZERO)
-            .medDagsats(finnDagsatsIkkeArbeidstaker(arbeidsforhold))
+            .medDagsats(finnDagsatsIkkeArbeidstaker(arbeidsforhold, utbetalingsgrad))
```

Comment:
Hører denne hjemme her? andelbuilder tar jo med utbetalingsgrad?

Reply:
Det er jo eit definisjonsspørsmål om dagsats skal vere gradert eller ugradert. Eg trur både k9-sak og fp-sak oppretter andel med gradert dagsats her. Det blir nok meir riktig å gjere det slik sidan logikken for å regne ut utbetalingsgrad i fp-sak og k9-sak ikkje er veldig god (f.eks 236% utbetalingsgrad på foreldrepenger som vi såg i produksjon denne veka).

Reply:
Men vi kan godt gjere om på dette. Vi må berre sørge for at FP og K9 beregner utbetalingsgrad riktig og at det henger sammen med det som faktisk utbetales.

Reply:
Det er et definisjonsspm om det skal være netto eller brutto dagsats - men det bør vel være konsekvent på tvers av kilder?
Utbetalingsgrad er forøvrig et langt fra veldefinert område ....

236% FP skyldes feil i gamle tilbaketrekksversjoner av BR der andeler ikke var kopiert rett - det oppstår ikke nye slike.

Reply:
For min del kan vi seie netto dagsats og så droppe utbetalingsgrad, evt berre opprette med 100% utbetalingsgrad. Vi er kun interessert i netto foreløpig.

Reply:
Det er i alle fall no konsekvent med korleis det blir gjort for FP og k9. Datakvaliteten for utbetalingsgrad er nok høgere hos infotrygd enn hos K9 og FP.

Reply:
Vi bruker utbetalingsgrad for YtelseAnvist til å identifisere og avstemme saker med overlapp -> regneark til enheter
Der bryr vi oss ikke om dagsats utover å sjekke > 0.
Utbetalingsgrad pr periode er OK for Infotrygd og Foreldrepenger (utenom om bugg i 5090/tilbaketrekk)
Den var ikke OK (les alltid 100%) for SVP  det er derfor jeg gjør en utregning. 
Men kan hende vi på sikt går over til å sjekke del 2 av overlapp - max 100% pr arbeidsforhold/giver